keywords/LiteServiOS.py

```diff
@@ -384,6 +392,27 @@ def stop(self):
             log_info("LiteServ is not running, so no need to stop")
         self._verify_not_running()
 
+    def _verify_running(self):
+        """
+        Return true if it is running or else false
+        Verifys that the endpoint does not return a 200 from a running service
+        """
+        try:
+            self.session.get("http://{}:{}/".format(self.host, self.port))
+        except ConnectionError:
+            # Expecting connection error if LiteServ is not running on the port
+            return False
+
+        return True
+
     def close_app(self):
         cur_dir = os.path.dirname(os.path.abspath(__file__))
         subprocess.check_output(["osascript", "{}/../utilities/sim_close_app.scpt".format(cur_dir)])
+
+    def open_app(self):
+        if(self.host == "localhost"):
+            output = subprocess.check_output(["ios-sim", "--devicetypeid", self.device, "launch", self.app_path, "--exit"])
+        else:
+            output = subprocess.check_output(["ios-deploy", "--justlaunch", "--bundle", self.app_path])
+        log_info("output of open app is {}".format(output))
```

Comment:
Is this a debug log? If not, rephrase the log message to some thing like "App launched with {}".format(output))